controller/userController.js

```diff
@@ -9,6 +9,17 @@ module.exports = {
   signup: async (req, res) => {
     const { uuid, nickName } = req.body;
 
+    const searchUuidResult = await sequelize.query(`SELECT uuid
+      from USER
+      where uuid like '%${uuid}%'`);

```

Comment:
여기서 혹시 같은 값을 찾지 않고 like 로 찾는 이유가 있을까요??
```suggestion
SELECT uuid FROM USER WHERE uuid = '%${uuid}%;'
```
이렇게 사용하시는게 좀 더 맞을거 같은데 어떠세요??ㅎㅎ

Reply:
아 like말고 이렇게 바로 찾기도 가능하군요!! ㅎㅎ
감사합니다~! 수정해서 머지할게요!!!